### ISSUE TITLE
Invert RequiredBy relationship in tomasys

### ISSUE THREAD

**Issue by Rezenders** [meta-control/mc_mros_reasoner#155]
The requiredby relationship is the only relationship in passive voice. It states that a component is required by function designs. IMO it should be the other way around, function designs require components

**Comment by chcorbato** [comment]
I agree. This is probably due to a quick hack and not using the TOMASys model for components in the implementation of MROS with SystemModes.

We need to review TOMASys model to provide a cleaner mapping into SystemModes
